src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigNumberOne = Big(numberOne);
+  const bigNumberTwo = Big(numberTwo);
+
+  let result = {};
+  let total = 0;
+
+  switch (operation) {
+    case '÷':
+      total = bigNumberOne / bigNumberTwo;
+      result = { total, next: total, operation };
+      break;
+    case '-':
+      total = bigNumberOne - bigNumberTwo;
+      result = { total, next: total, operation };
+      break;
+    case '+':
+      total = bigNumberOne + bigNumberTwo;
+      result = { total, next: total, operation };
+      break;
+    case 'X':
+      total = bigNumberOne * bigNumberTwo;
+      result = { total, next: total, operation };
+      break;
+    default:
+      result = { bigNumberOne, bigNumberTwo, operation };
+  }
```

Comment:
- [ ] the `Modifier (%)` operation should be present in the `operate.js` module
Screenshot from pathwright instructions on implementing the `operate.js` module 👇🏾 
<img width="807" alt="Screen Shot 2021-02-10 at 8 30 11 PM" src="https://user-images.githubusercontent.com/48804258/107561400-debac700-6bb4-11eb-85c3-c62b7e5c1e7e.png">